ci/values/k3s.yaml

```diff
@@ -27,3 +23,6 @@ redis:
   master:
     persistence:
       size: 1Gi
+zookeeper:
+  persistence:
+    size: 1Gi
```

Comment:
Is this a breaking change for users with defaults? Will this cause trouble for them?

Reply:
> Is this a breaking change for users with defaults? Will this cause trouble for them?

This is not, it is only an override for the `k3s` test

> Also this seems undocumented in the main values.yaml file.

Correct, we do not expose all the variables from subcharts (we probably should)

Reply:
Ah, didn't notice the CI dir here.